tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Response.cs

```diff
@@ -15,26 +15,35 @@ public class Response
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public List<string> ResponseErrors { get; set; }
 
+    [JsonPropertyName("next_steps")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public List<string>? NextSteps { get; set; }
+
     protected string ToString(StringBuilder value)
     {
         return ToString(value.ToString());
     }
 
     protected string ToString(string value)
     {
-        List<string> errors = [];
+        List<string> messages = [];
         if (!string.IsNullOrEmpty(ResponseError))
         {
-            errors.Add("[ERROR] " + ResponseError);
+            messages.Add("[ERROR] " + ResponseError);
         }
         foreach (var error in ResponseErrors ?? [])
         {
-            errors.Add("[ERROR] " + error);
+            messages.Add("[ERROR] " + error);
+        }
+
+        foreach (var nextStep in NextSteps ?? [])
+        {
+            messages.Add("[NEXT STEPS] " + nextStep);
         }
 
-        if (errors.Count > 0)
+        if (messages.Count > 0)
         {
-            value = string.Join(Environment.NewLine, errors);
+            value = string.Join(Environment.NewLine, messages);
```

Comment:
do we think the next steps may always be populated, we just only want it to output when the failure happens? In which case this would need to change

Reply:
I think in both success and failure cases we should output next steps if it's populated, that way each sub-class doesn't have to stringify next steps.